tests/scale/test_load.py

```diff
@@ -0,0 +1,170 @@
+"""
+A way to launch numerous Jenkins masters and jobs with pytest.
+
+From the CLI, this can be run as follows:
+    $ PYTEST_ARGS="--masters=3 --jobs=10" ./test.sh -m scale jenkins
```

Comment:
Is there any reason we are using`pytest` instead of **python** for this? (That's like using junit instead of java -jar).

Reply:
Agree with @colin-msphere.  @orsenthil We're avoiding building a lot of new tooling in the interests of expediency right now.